src/Main.cc

```diff
@@ -213,8 +214,12 @@ int main(int argc, char* argv[])
 {
   // This avoids problems with locale setting dependent
   // C functions like strtod (e.g. 0.5 vs 0,5).
-  setenv("LC_ALL", "en_US.UTF-8", /* override */ 1);
-  setenv("LANG", "en_US.UTF-8", /* override */ 1);
+  std::array locales = {"en_US.UTF-8", "C.UTF-8", "C"};
+  for (auto locale : locales) {
+    if (std::setlocale(LC_ALL, locale) != nullptr) {
```

Comment:
Why LC_ALL? Shouldn't we just set LC_NUMERIC. I'm adding translations of the GUI at the moment, and I think this might make it difficult to respect the user's locale info.

Reply:
We were already setting LC_ALL so this is no change (I was also setting LANG before too).  

I don't know if there are any other cases of concern.  It would also be a bit odd to have a mix of LC settings.  What GUI issues do you see with this?

Reply:
I think this concern is separate from the change here.  If we need to change this for other reasons we can address that as part of your GUI work.